saleor/payment/gateways/np_atobarai/errors.py

```diff
@@ -0,0 +1,32 @@
+from typing import Iterable, List
+
+TRANSACTION_REGISTRATION = "TR"
+TRANSACTION_CANCELLATION = "TC"
+TRANSACTION_CHANGE = "CH"
+FULFILLMENT_REPORT = "FR"
+
+
+def add_action_to_code(action: str, error_code: str) -> str:
+    return f"{action}#{error_code}"
+
+
+def get_error_messages_from_codes(action: str, error_codes: Iterable[str]) -> List[str]:
+    return [add_action_to_code(action, code) for code in error_codes]
```

Comment:
Couldn't `add_action_to_code` be inline here?

Reply:
It's also used in `api.py` module